src/olympia/editors/management/commands/auto_approve.py

```diff
@@ -23,6 +24,7 @@
 
 class Command(BaseCommand):
     help = 'Auto-approve add-ons based on predefined criteria'
+    lock_name = 'auto-approve'  # Name of the atomic_lock() used.
```

Comment:
It's only used in the command, but I guess it could be moved to a constant at the top of the file, yes. I'll do that.